src/client/thirdParty/Amplitude.tsx

```diff
@@ -3,7 +3,10 @@ import { usePlaygroundSignals } from '../../app/playground/hooks/usePlaygroundSi
 import { FunctionComponent } from 'react';
 
 const AMPLITUDE_INGRESS_PROXY = 'https://demo.fingerprint.com/ampl-api/2/httpapi';
-const EVENT_TYPE = 'Demo Page Viewed';
+const AMPLITUDE_EVENT = {
+  DEMO_PAGE_VIEWED: 'Demo Page Viewed',
+  DEMO_ASK_AI_HELP_CHOSEN: 'Demo Ask AI Help Chosen',
```

Comment:
@hba-fingerprint I pushed a commit to apply some subjective code preferences. There are still inconsistencies between the implementation and the ticket: 

* Ticket says the event name should be `Demo Ask AI Get Help Chosen`
* Ticket does not specify `visitorId` as a required property. 

Is this expected? If the requirements changed can you update the ticket description to reflect that? cc @Zaciesh

Reply:
Ok, just had another look, following what we have on amplitude already, the method name is really Demo Ask AI Help Chosen, without Get, so the ticket might be off. 

VisitorId is indeed non-required and is being received as unknown. I'll remove it.

Reply:
Anything marked as "Unexpected" means that it is not expected based on the tracking plan defined by the analyst in the tracking plan. Anything unexpected in production will not be ingested. Instead, it will trigger alert that highlights that we try to pass something we do not expect on the analytics end.

Reply:
That sounds good. I'm removing visitorId from it and the name of the other properties were already updated. Thanks!

Reply:
Thanks, both! resolving